lxd/firewall/drivers/drivers_xtables.go

```diff
@@ -1155,11 +1155,30 @@ func (d Xtables) matchEbtablesRule(activeRule []string, matchRule []string, dele
 			continue
 		}
 
-		// Mangle to line up with different versions of ebtables.
-		active := strings.Replace(activeRule[i], "/ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff", "", -1)
-		match := strings.Replace(matchRule[i], "/ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff", "", -1)
-		active = strings.Replace(active, "fe80::/ffc0::", "fe80::/10", -1)
-		match = strings.Replace(match, "fe80::/ffc0::", "fe80::/10", -1)
+		active := activeRule[i]
+		match := matchRule[i]
+
+		// Convert masks to cidr ranges (or remove if full mask) to line up with different versions of ebtables.
```

Comment:
Can you expand this comment here explaining what this part is doing, showing examples. Because previously the code itself was very simple and provided its own examples effectively, whereas now I find this very difficult to understand and couldn't start understand what its aiming to do without having the previous version there to help :)

Reply:
I've rewritten this logic and added comments and I think it's a lot clearer. Let me know what you think.